internal/config/config.go

```diff
@@ -1,16 +1,27 @@
 package config
 
 import (
-	"github.com/go-playground/validator/v10"
-	"gopkg.in/yaml.v3"
 	"os"
+
 	"viktig/internal/services/forwarder"
 	"viktig/internal/services/http_server"
+	"viktig/internal/services/http_server/handlers/metrics_handler"
+
+	"github.com/go-playground/validator/v10"
+	"gopkg.in/yaml.v3"
 )
 
 type Config struct {
-	ForwarderConfig  *forwarder.Config   `yaml:"forwarder_config" validate:"required"`
-	HttpServerConfig *http_server.Config `yaml:"http_config" validate:"required"`
+	ForwarderConfig  *forwarder.Config       `yaml:"forwarder_config" validate:"required"`
+	HttpServerConfig *http_server.Config     `yaml:"http_config" validate:"required"`
+	TempConfig       *TempConfig             `yaml:"temp_config" validate:"required"`
+	MetricsConfig    *metrics_handler.Config `yaml:"temp_config"`
```

Comment:
```
panic: duplicated key 'temp_config' in struct config.Config [recovered]
```
Also consider making required. I'm not sure what will happen if this is not specified, but if that will mean no auth for `/metrics` then that's a security issue